src/screens/TripSettings/TripSettings.tsx

```diff
@@ -14,13 +35,326 @@ type TripItineraryScreenNavigationProp = CompositeNavigationProp<
+                                    containerStyle={styles.buttonContainer}
+                                    buttonStyle={styles.submitButton}
+                                    title={t("screens.tripSettings.submitUpdate")}
+                                    titleStyle={{
+                                        color: "black",
+                                        fontSize: 25,
+                                    }}
+                                    onPress={() => handleSubmit()}
+                                    loading={loadingUpdate}
+                                />
+                            )}
+                        </>
+                    )}
+                </Formik>
+                <Text h3 style={{ marginBottom: 15 }}>
+                    {t("screens.tripSettings.members")}
+                </Text>
+                {data.trip.members.map((member, i) => (
+                    <ListItem key={i} bottomDivider>
+                        <Avatar
+                            rounded
+                            containerStyle={{
+                                backgroundColor: stc(member.user.id),
+                                marginRight: 5,
+                            }}
+                            size="medium"
+                            title={member.user.displayName.charAt(0)?.toUpperCase() ?? "?"}
+                        />
+
+                        <ListItem.Content>
+                            <ListItem.Title>{member.user.displayName}</ListItem.Title>
+                            {member.role === "ADMIN" && (
+                                <ListItem.Subtitle>
+                                    {member.role.charAt(0).toUpperCase() +
+                                        member.role.toLowerCase().slice(1)}
+                                </ListItem.Subtitle>
+                            )}
+                        </ListItem.Content>
+                        {settingsMode === Mode.EDIT &&
+                            data.trip.admin.id === currentUser?.id &&
+                            member.user.id !== currentUser?.id && (
+                                <ListItem.Chevron
+                                    onPress={() => handleRemoveMember(member.user.id)}
+                                    name={loadingRemoveMember ? "spinner" : "trash-alt"}
+                                    size={20}
+                                    color="#e03030"
+                                    type="font-awesome-5"
+                                />
+                            )}
+                    </ListItem>
+                ))}
+                {errorLeave && <Text style={styles.errorText}>{errorLeave.message}</Text>}
+                {errorRemoveMember && (
+                    <Text style={styles.errorText}>{errorRemoveMember.message}</Text>
+                )}
+                {data.trip.admin.id !== currentUser?.id && data.trip.members.length > 1 && (
```

Comment:
```suggestion
                {data.trip.admin.id !== currentUser?.id && data.trip.members.length > 1 && settingsMode === Mode.EDIT && (
```
depends again if a member can only leave when being in edit mode or not